.travis.yml

```diff
@@ -6,53 +6,56 @@ env:
         - use_coveralls=true
     matrix:
         - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
+          CONTAINER_CMD=docker
```

Comment:
Please place this variable in `global` section.

Reply:
Done, thanks!